### DISCUSSION TITLE
源码构建镜像成功，但启动报错：No module named 'paddle.nn.layer.layers

### DISCUSSION THREAD

**Discussion by Greenarrow1224** [PaddlePaddle/PaddleOCR#12482] [Q&A]
请提供下述完整信息以便快速定位问题/Please provide the following information to quickly locate the problem

- 系统环境/System Environment：linux 
- 版本号/Version：Paddle：2.4.0  PaddleOCR：release/2.7 分支源码 问题相关组件/Related components：
- 运行指令/Command Code：hubserving  方式 dokcer 构建镜像
- 完整报错/Complete Error Message：No module named 'paddle.nn.layer.layers

我们提供了AceIssueSolver来帮助你解答问题，你是否想要它来解答(请填写yes/no)?/We provide AceIssueSolver to solve issues, do you want it? (Please write yes/no): yes

请尽量不要包含图片在问题中/Please try to not include the image in the issue.

**Comment by SWHL** [comment]
如果问题仍然存在，请拉取仓库main分支代码，再次尝试
